src/topology/algebra/ordered.lean

```diff
@@ -96,6 +96,22 @@ lemma closure_lt_subset_le [topological_space β] {f g : β → α} (hf : contin
   closure {b | f b < g b} ⊆ {b | f b ≤ g b} :=
 by { rw [←closure_le_eq hf hg], exact closure_mono (λ b, le_of_lt) }
 
+lemma continuous_within_at.closure_le [topological_space β]
```

Comment:
To obtain `f x ≤ g x` at a given `x` in the closure of `s`, you only need the continuity of `f` and `g` at this specific `x`. So, I think a statement involving one fixed `x` and assumptions `continuous_within_at f s x`, `continuous_within_at g s x` and `x ∈ closure s` would be more natural.

Reply:
Good point. I fixed that. But maybe I'm having too much fun with filters in the new proof...